templates/components/category/sidebar.html

```diff
@@ -1,20 +1,22 @@
-<nav>
-    {{#if category.subcategories}}
-    <div class="sidebarBlock">
-        <h5 class="sidebarBlock-heading">{{category.name}}</h5>
-        <ul class="navList">
-            {{#each category.subcategories}}
-            <li class="navList-item">
-                <a class="navList-action" href="{{url}}" alt="{{name}}" title="{{name}}">{{name}}</a>
-            </li>
-            {{/each}}
-        </ul>
-    </div>
-    {{/if}}
+<aside class="page-sidebar" id="faceted-search-container">
+    <nav>
+        {{#if category.subcategories}}
+        <div class="sidebarBlock">
+            <h5 class="sidebarBlock-heading">{{category.name}}</h5>
+            <ul class="navList">
+                {{#each category.subcategories}}
+                <li class="navList-item">
+                    <a class="navList-action" href="{{url}}" alt="{{name}}" title="{{name}}">{{name}}</a>
+                </li>
+                {{/each}}
+            </ul>
+        </div>
+        {{/if}}
 
-    {{#if category.faceted_search_enabled}}
-        {{> components/faceted-search/index category}}
-    {{else}}
-        {{> components/category/shop-by-price shop_by_price=category.shop_by_price category_url=category.url}}
-    {{/if}}
-</nav>
+        {{#if category.faceted_search_enabled}}
+            {{> components/faceted-search/index category}}
+        {{else}}
+            {{> components/category/shop-by-price shop_by_price=category.shop_by_price category_url=category.url}}
+        {{/if}}
+    </nav>
+</aside>
```

Comment:
verified this is only used by the file im changing - also not sure why the diff is showing like this all i did was move the aside tag in here